temporal-server.yaml

```diff
@@ -43,30 +43,62 @@ pipeline:
   - uses: strip
 
 subpackages:
+  - name: temporal-server-compat
+    description: "Compat package for temporal-server"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/local/bin
+          ln -s /usr/bin/temporal-server "${{targets.subpkgdir}}"/usr/local/bin
+      - uses: strip
+
   - name: temporal-cassandra-tool
     description: "temporal-cassandra-tool"
     pipeline:
       - runs: |
-          mkdir -p "${{targets.subpkgdir}}"/usr/bin
+          mkdir -p "${{targets.subpkgdir}}"/usr/local/bin
           install -Dm755 temporal-cassandra-tool "${{targets.subpkgdir}}"/usr/bin/temporal-cassandra-tool
       - uses: strip
 
+  - name: temporal-cassandra-tool-compat
+    description: "Compat package for temporal-cassandra-tool"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/local/bin
+          ln -s /usr/bin/temporal-cassandra-tool "${{targets.subpkgdir}}"/usr/local/bin
+      - uses: strip
+
   - name: tdbg
     description: "tdbg"
     pipeline:
       - runs: |
-          mkdir -p "${{targets.subpkgdir}}"/usr/bin
+          mkdir -p "${{targets.subpkgdir}}"/usr/local/bin
           install -Dm755 tdbg "${{targets.subpkgdir}}"/usr/bin/tdbg
       - uses: strip
 
+  - name: tdbg
```

Comment:
I personally think, having two sub packages of same names is not feasible, we can add something like compat to them while doing symlinks, @rawlingsj may have more inputs on them? Similar for others

Reply:
having compact makes sense, fixed

Reply:
A minor nit, I think it should be compat (short for compatibility). Also I see the scanning of CVE fails, will be good to see if it’s fixable or we have to need an advisory for that, thanks for doing it!